docs/pages/includes/database-access/db-audit-events.mdx

```diff
@@ -0,0 +1,38 @@
+You can view database session activity in the Audit Log
+and after the session is uploaded retrieve with the command `tsh play`.
+
+```code
+# Database session ID will be in a UUID format (ex: 307b49d6-56c7-4d20-8cf0-5bc5348a7101)
+# See the audit log to get a session ID with a key of sid
```

Comment:
These two lines aren't code. I would move them out of the code block. Then you can put `sid` in backticks.

Reply:
Please see update.